src/Commands/WaContent.php

```diff
@@ -125,6 +128,82 @@ private function createPost($waContent)
     private function handleTranslation($postId, $waContent)
     {
         LanguageProvider::setPostLanguage($postId, $waContent->translation->locale ?? 'da');
+
+        //if this is not the master translation, just return
+        if (!$waContent->translation->is_master || !isset($waContent->translation)) {
+            return;
+        }
+
+        $postTranslations = collect($waContent->translation->translation_ids);
+
+        if (empty($postTranslations)) {
+            WP_CLI::warning('found no translations. returning.');
+            return;
+        }
+
+        WP_CLI::success(sprintf('found the following translations: %s', json_encode($postTranslations, JSON_PRETTY_PRINT)));
+        $translationPostIds = $postTranslations->map(
+            function ($translation_id, $locale) use ($waContent) {
+                if (null == (env($endpoint = strtoupper('WHITEALBUM_ENDPOINT_'.$locale)))) {
+                    WP_CLI::warning(sprintf('%s has not been defined in your ENV file.', $endpoint));
+                    return;
+                }
+                $repository = new ContentRepository(env($endpoint));
```

Comment:
Then you can reuse the `$endpoint` from above, instead of re-fetching from `.env`?